src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -42,42 +42,9 @@
           />
         </v-col>
       </v-row>
-
-      <!-- Expiry Date -->
-      <v-row
-        class="mt-4"
-        no-gutters
-      >
-        <v-col
-          cols="12"
-          sm="3"
-        >
-          <label>Expiry Date</label>
-        </v-col>
-
-        <v-col
-          cols="12"
-          sm="9"
-          class="pl-8"
-        >
-          <DatePickerShared
-            id="expiry-date"
-            ref="expiryDateRef"
-            title="Expiry Date (Optional)"
-            :nudgeRight="40"
-            :nudgeTop="85"
-            hint="The date the authorization expires"
-            :persistentHint="true"
-            :initialValue="authorization.expiryDate"
-            :inputRules="getShowErrors ? expiryDateRules: []"
-            :minDate="getCurrentDate"
-            @emitDateSync="authorization.expiryDate = $event"
-          />
-        </v-col>
-      </v-row>
     </v-form>
 
-    <v-divider class="mt-6 mb-8" />
+    <v-divider class="mt-4 mb-8" />
```

Comment:
Fixed whitespace above divider so it looks good.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/8eade94c-2643-4adc-aeda-3cea3aea9101)